sentry-android-core/src/test/java/io/sentry/android/core/ManifestMetadataReaderTest.kt

```diff
@@ -46,35 +46,35 @@ class ManifestMetadataReaderTest {
     fun `applyMetadata reads sampleRate from metadata`() {
         // Arrange
         val options = SentryAndroidOptions()
-        val expectedSampleRate = 0.99
+        val expectedSampleRate = 0.99f
 
         val bundle = Bundle()
         val mockContext = ContextUtilsTest.mockMetaData(metaData = bundle)
-        bundle.putDouble(ManifestMetadataReader.SAMPLE_RATE, expectedSampleRate)
```

Comment:
`m`: Maybe it makes sense to add keep a test with a double value and make sure our code works properly. The `metadata` is a `Bundle`, so it could contain a `Double` for `SAMPLE_RATE`.

Reply:
nope, this is the way how to mock a `Bundle` meta-data, we always read from the manifest file which can never be a `Double`